src/api/app/controllers/webui/request_controller.rb

```diff
@@ -190,14 +190,7 @@ def delete_request_dialog
   def delete_request
     req = nil
     begin
-      BsRequest.transaction do
-        opts = { target_project: params[:project], description: params[:description] }
-        opts[:target_package] = params[:package] if params[:package]
-        opts[:target_repository] = params[:repository] if params[:repository]
-        req = BsRequest.build_from_params(:delete, opts)
-        req.save!
-      end
-
+      req = BsRequest.create_from_params!(request_attributes(:delete))
```

Comment:
Why not just

```
req = BsRequest.new(attributes[:request]).bs_request_actions.build(attributes[:request_action]).save!
```

instead of creating a new method for it? People know what `.new` and `.build` do. They have to look up what `.create_from_params!` means.

Even better would be if `BsRequest` would `accepts_nested_attributes_for :bs_request_action`...